src/modules/closedcaptions.json

```diff
@@ -0,0 +1,403 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+        "title": "ClosedCaptions",
+        "description": "A module for managing closed-captions Settings.",
+        "version": "0.0.0"
+    },
+    "methods": [
+        {
+            "name": "onRequestClosedCaptionsSettings",
+            "tags": [
+                {
+                    "name": "events",
+                    "x-provides": [
+                        "xrn:firebolt:capability:settings:closedcaptions"
+                    ],
+                    "x-response": {
+                        // TODO: rationalize this with the `result` below...
+                        "$ref": "#/components/schemas/ClosedCaptionsSettingsProviderResponse"
+                    }
+                }
+            ],
+            "summary": "Dispatches a request for the current closed-captions settings to the closed-captions settings provider.",
+            "params": [],
+            // TODO: should this go away?
+            "result": {
+                "name": "settings",
+                "schema": {
+                    "$ref": "https://meta.comcast.com/firebolt/accessibility#/definitions/ClosedCaptionsSettings"
```

Comment:
This is not right. The result for the provider subscription call onRequestClosedCaptionsSettings would be the event that triggers asking for closedCaptionSettings.

In terms of javascript:
```
class CCProvider {
   closedCaptionsSettings(request, responder) {
      responder(settings)
   }
}
```

So "result" here would be "request" in that provider function. Guessing we don't need any more information in the request so it can just be empty.
"settings" would be the actual ClosedCaptionsSettings object, which we call `responder` with.

Reply:
Right, thanks! I had a note to go remember how that works.

Reply:
Okay, fixed.